contracts/contracts/NoLossDao_v0.sol

```diff
@@ -348,26 +348,24 @@ contract NoLossDao_v0 is Initializable {
     //   'Time that this iteration has ended incremented ',
     //   proposalDeadline
     // );
-    if (topProject[proposalIteration] != 0) {
-      // Do some asserts here for safety...
-
+    if (proposalIteration > 0) {
       // Only if last winner is not withdrawn (i.e. still in cooldown) make it active again
       if (
         state[topProject[proposalIteration.sub(1)]] == ProposalState.Cooldown
       ) {
         state[topProject[proposalIteration.sub(1)]] = ProposalState.Active;
       }
-      // Only if they haven't withdrawn, put them in cooldown
```

Comment:
These are the kind of comments that I feel don't add much, and can super easily go stale and say something different to the code.

Its a bit like writing:
"I am adding five to three here" above "5 + 3" in a maths document :D